packages/documentation-v7/src/stories/foundation/elevation/elevation.docs.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import { BADGE } from '../../../../.storybook/constants';
+import { ElevationTile } from './elevation.components';
+import './elevation.styles.scss';
+
+<Meta
+  title="Foundations/Elevation"
+  parameters={{ badges: [BADGE.NEEDS_REVISION] }}
+/>
+
+# Elevation
+
+<p className="lead">Elevation uses box shadows to give the impression of a distance between two elements along the z axis.</p>
+
+The higher the elevation, the larger and softer the shadow, expressing a greater distance between an element and the one behind it.
+
+<div className="elevation-container">
+  {[1, 2, 3, 4, 5].map((elevationLevel, i) => (
```

Comment:
We could get the amount of elevation levels from a elevation.module.scss.
What do you think?